sql/core/src/test/scala/org/apache/spark/sql/streaming/TransformWithStateSuite.scala

```diff
@@ -784,6 +787,146 @@ class TransformWithStateSuite extends StateStoreMetricsTest
       }
     }
   }
+
+  private def fetchColumnFamilySchemas(
```

Comment:
Could we also add some unit tests also that can check for the schema file format creation / ser-de etc ?